js/battle.js

```diff
@@ -553,7 +556,7 @@ var Pokemon = (function () {
 			if (plaintext) {
 				name += ' (' + this.species + ')';
 			} else {
-				name += ' <small>(' + this.species + ')</small>';
+				name = '<span class="pokemon" title="' +  this.species + '">' + name + ' <small>(' + this.species + ')</small>' + '</span>';
```

Comment:
Tooltips are not part of the room, unfortunately. They're also kind of very messy code... Do whatever you feel is best.